awx/api/views/__init__.py

```diff
@@ -2143,6 +2152,11 @@ def post(self, request, *args, **kwargs):
         return super(InventorySourceNotificationTemplatesAnyList, self).post(request, *args, **kwargs)
 
 
+class InventorySourceNotificationTemplatesStartedList(InventorySourceNotificationTemplatesAnyList):
```

Comment:
So it's kind of interesting to me that the views defined *here* all subclass the `All` implementation, and *just* special a _different_ `relationship`, but (see comment above...